tools/cloudharness_utilities/deployment-configuration/codefresh-template-prod.yaml

```diff
@@ -30,7 +30,7 @@ steps:
     working_directory: .
     commands:
       - bash cloud-harness/install.sh
-      - harness-deployment . cloud-harness -t ${{DEPLOYMENT_TAG}} -d ${{DOMAIN}} -r ${{REGISTRY}} -rs ${{REGISTRY_SECRET}} -e prod
+      - harness-deployment . cloud-harness -t ${{DEPLOYMENT_TAG}} -d ${{DOMAIN}} -r ${{REGISTRY}} -rs ${{REGISTRY_SECRET}} -e $ENV
```

Comment:
@zsinnema please note this: with this template change we are not anymore forced to update templates to only change the environment

Reply:
@filippomc I've added the deployment label because this will break the current deployments in Codefresh (then don't have the ENV variable set)

Reply:
they won't, it's not required so old deployments without the $ENV variable will work as usual